Mesh_3/include/CGAL/Mesh_3/tet_soup_to_c3t3.h

```diff
@@ -362,7 +362,7 @@ bool build_triangulation_from_file(std::istream& is,
 
   // grab the vertices
   int dim;
-  int nv, nf, ntet;
+  int nv, nf, ntet, useless;
```

Comment:
Please have a look at the `.mesh` writer to find out what these `useless` values actually are.
Then
- change their names,
- amend this commit (51bff2526e48faa14d757252075392fbd19a2e42) with the updated variable names, and a more explicit commit message
thanks